### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by Vaishnavi-Thiyagaraj**

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -1,11 +1,5 @@
...
-const db = require('./db/models/index.js');
 
 const PORT = process.env.PORT || 3004;
+const app = require('./app.js');
 
-const app = express();
-app.use(express.static('public'));
-app.get('/app.bundle.js', (req, res) => {
-  res.sendFile('public/app.bundle.js', { root: '../' });
-});
```
Comment (MattRaetz) [inline server/index.js:10]: It's not a problem at all, but I'm curious why you split the previous section into a separate file?

File: server/app.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const service = require('./controllers.js');
+
+const app = express();
+
+app.use(express.static('public'));
+app.get('/app.bundle.js', (req, res) => {
+  res.sendFile('public/app.bundle.js', { root: '../' });
+});
```
Comment (MattRaetz) [inline server/app.js:9]: This is interesting and I need to look into more on how this works as it's fairly different than mine.

**Review by MattRaetz — APPROVED** [review]
I don't see any issues. Looks good.

File: server/index.js (hunk 1 above)
Comment (Vaishnavi-Thiyagaraj) [reply server/index.js:10]: I want to keep my routes separately so i can easily import and test it without messing my app server

**Pull request merged**